README.md

```diff
@@ -230,12 +230,21 @@ curl -F "data=@<Path to first evaluation File>" -F "data=@<Path to second evalua
 
 ### How to Install
 
-If you would like to change Heimdall to your needs, Heimdall has 'Development Mode' you can use, where if you make changes to the code, the app will automattically rebuild itself and use those changes. Please note that you should *not* run development mode when deploying Heimdall for general usage. To get started on a Debian-based distribution, follow these steps:
+If you would like to change Heimdall to your needs, Heimdall has 'Development Mode' you can use, where if you make changes to the code, the app will automatically rebuild itself and use those changes. Please note that you should *not* run development mode when deploying Heimdall for general usage. To get started on a Debian-based distribution, follow these steps:
 
-1. Install system dependencies:
+1. Install system dependencies with your system's package manager.
 
+   Debian/Ubuntu:
+
+   - ```bash
+     sudo apt install postgresql nodejs=16 nano git
```

Comment:
`sudo apt install postgresql nodejs=16 nano git` returns error on Ubuntu v20 vm test. NPM officially recommends users to install npm/nodejs using node version managers, so if we assume the user has nvm/n/etc., couldn't we just use the version manager to specify the nodejs version?

Reply:
Tested; this doesn't actually work on Debian. Adding some guidance for installing the right nodejs version on Ubuntu.

We'll have to add some guidance on Windows installation too, but likely as a separate PR.